docs/input.html

```diff
@@ -0,0 +1,183 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css"
+      integrity="sha512-1ycn6IcaQQ40/MKBW2W4Rhis/DbILU74C1vSrLJxCq57o941Ym01SwNsOMqvEBFlcgUa6xLiPY/NS5R+E6ztJQ=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="stylesheet" href="../lib/index.css" />
+    <link rel="stylesheet" href="/css/docs.css" />
+    <title>Input</title>
+  </head>
+  <body>
+    <nav
+      class="d-flex items-center topnav border-b-primary-1 index-999 bg-secondary"
+    >
+      <ul>
+        <li>
+          <a href="/" class="navlink">FlixU</a>
+        </li>
+      </ul>
+      <ul class="d-flex ml-auto">
+        <li>
+          <a href="/docs/alert.html" class="navlink">Documentation</a>
+        </li>
+        <li>
+          <a href="https://github.com/Debarshi95/FlixUI" class="navlink">
+            Github</a
+          >
+        </li>
+      </ul>
+    </nav>
+    <main class="min-h-screen body-container">
```

Comment:
Add comments in your html cod

Reply:
Thanks but are comments really necessary?